cmd/pmm-managed-starlark/main_test.go

```diff
@@ -0,0 +1,157 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/percona/pmm-managed/services/checks"
+)
+
+const (
+	invalidStarlarkScriptStderr = "Error running starlark script: thread invalid starlark script: failed to execute function check: function check accepts no arguments (1 given)"
+	invalidVersionStderr        = "Error running starlark script: unsupported check version: 5"
+	memoryConsumingScriptStderr = "fatal error: runtime: out of memory"
+)
+
+var validQueryActionResult = []map[string]interface{}{
+	{"Value": "5.7.30-33-log", "Variable_name": "version"},
+	{"Value": "Percona Server (GPL), Release 33, Revision 6517692", "Variable_name": "version_comment"},
+	{"Value": "x86_64", "Variable_name": "version_compile_machine"},
+	{"Value": "Linux", "Variable_name": "version_compile_os"},
+	{"Value": "-log", "Variable_name": "version_suffix"},
+}
+
+func TestRunChecks(t *testing.T) {
+	testCases := []struct {
+		version      uint32
```

Comment:
Version is constant in all tests, so let's remove it

Reply:
we actually have a test case where the script exits with status 1 because of a wrong version.

Reply:
Do we check version in this package? No. So why we test it there?

We exit with status code 1 on any "Error running starlark script" error. The code for that is in this package. So it is enough to test it once.

Reply:
i somehow missed this comment, will remove version and that test case

Reply:
Please don't :)

Reply:
i just did